feat(global): :sparkles: passing prop in LinearProgress

PritamBag opened the pull request:
## Description
I updated the code, where pass two props (showLabel, labelWidth), if the showLabel prop is passed as true then a label is displayed along with the LinearProgressbar. Also, the labelWidth prop is responsible for label width which is by default 35px, also users can change this width as they need by passing a value in the labelWidth prop. 
Ref: https://github.com/wrappid/native-web/issues/114

## Related Issues

https://github.com/wrappid/core/issues/332 

## Testing
I pass those props in the code of this(CoreLinearProgress.docs.js) file, and the code works properly.  

## Checklist

- [X] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [X] My code follows the project's style guidelines and best practices.
- [X] I have updated the documentation if necessary.
- [X] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts in my branch.

## Screenshots (if applicable)

## Additional Notes

## Reviewers

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

PritamBag committed:
feat(global): :sparkles: passing prop for label

passing props in LinearProgress, if user pass this prop then label is displayed in UI

Ref: #114

techoneel commented on package/nativeComponents/feedback/NativeLinearProgress.js:
```diff
@@ -1,8 +1,32 @@
+  const { children, showPercentage, ...rest } = props;
+  const boxWidth = "35px";
+
+  return (
```
This will happen if the label needs to be shown.
What if we don't want to show the label then the simple SCLinearProgress needs to be returned.

techoneel reviewed (changes requested):
Please do the needful

PritamBag committed:
feat(global): :sparkles: updated code of NativeLinearProgress

rewrite the code as mentation in comment(if else), also pass width using labelWidth props

Ref: #114

techoneel reviewed (approved):
good to go
